pkg/kubelet/container/helpers.go

```diff
@@ -46,7 +46,7 @@ type HandlerRunner interface {
 // RuntimeHelper wraps kubelet to make container runtime
 // able to get necessary informations like the RunContainerOptions, DNS settings, Host IP.
```

Comment:
Update the comment to reflect the change.

Reply:
Thanks for the quick review. This comments seems correct to me, as `GetClusterDNS()` will still return DNS settings :)

Reply:
Oops... you're right.. didn't notice the comment was for the entire `RuntimeHelper`.